### ISSUE TITLE
[Bug] Weird app behaviour when selecting points

### ISSUE THREAD

**Issue by collectorgeneral** [redfish64/TinyTravelTracker#47]
General Information:
App details:
Version: 1.1.15; Source: F-droid
OS: Cyanogen OS 13.1.2 (Android 6.0.1)
Device: Oneplus One (aka A0001 aka Bacon)

Description:
I was looking at some points today and I touched the screen. The info panel comes up, and I accidentally touch the time information. It only shows those points now. I try to use the scroller below to get it to display all the points, but it only shows a few points at a time. I go to the date range menu and try it again, but to no avail. It is only displaying a few points at a time and scrolling does not seem to work as intended. I also encountered a crash, though I am not sure if it is fixed in the versions after. I am attaching the crash report.
[20160907_TTT_Log.txt](https://github.com/redfish64/TinyTravelTracker/files/460108/20160907_TTT_Log.txt)

**Comment by collectorgeneral** [comment]
@redfish64 
Update:

Actually, never mind. I just realised now that you can 'drag' the time bar up and down to increase/decrease time scale. It was not intuitive at first, but I figured that out after messing around with it. The crash still occurs though, not sure why. I am leaving this issue open for you to figure out the crash.

This reminds me, maybe we should have a small wiki page to explain such features? I can write the content for you.

**Comment by israel-lugo** [comment]
I'm also seeing a crash when I scroll up (to tighten the time span), while I have a point selected and am looking at its information. I.e.:

1. Select an existing point in the past, to view its information
1. Scroll up on the time window, to tighten the time span
1. The application crashes and exits

I would guess that the problem may be related to the time span going to an interval that doesn't include the selected point, but can't be sure.

A similar crash happens when scrolling left or right on the time span a few times, instead of scrolling up. With the point selected, I scrolled left to a previous time and the point disappeared (as expected). Then I scrolled to the right and the point reappeared (with information). I repeated this 2 or 3 times, and the application crashed (while scrolling to the left).

One Plus 2, A2003
OxygenOS 2.2.1
Android 5.1.1

**Comment by collectorgeneral** [comment]
@israel-lugo 
Yup, seems pretty similar to what I did to get the crash. It is not a deal breaker, but it could be a tad bit annoying. Hopefully @redfish64  can figure out the issue.
